src/metadata/SDKCommandsMetadata.json

```diff
@@ -590,7 +590,7 @@
 	"importobjects": {
 		"name": "importobjects",
 		"usage": "",
-		"description": "Import custom objects from an account to your project.",
+		"description": "Imports custom objects from your NetSuite account to the SDF project. In account customization projects (ACP), if SuiteScript files are referenced in the custom objects you import, these files get imported by default.",
```

Comment:
Yes. I generated the metadata. However, I did not copy the whole generated one, because when I compared, in the latest metadata there were differences that come from another story. I thought those metadata changes will come from the merge request for that story (PDPDEVTOOL-1951).